### PR TITLE
closes #85

### LINKED ISSUE #85: Change the content text box to full markdown friendly textboxes that shows preview as you type.
On the new question and show question page, it would be nice to have a better textbox that allows for markdown friendly texts. and preview as you type

### PR DESCRIPTION
**Pull request by innocentamadi**
This PR removes tinyMCE textbox and replaces it with a regular textbox that accepts markdown and shows preview of the content typed in real time.

### CODE REVIEW

**Comment by andela-cdaniel** [comment]
The preview only shows up once you click outside of the `textarea`. I think it would be better if it shows the result in real time (as I am typing) rather than when the `textarea` loses focus.

**Comment by andela-iamadi** [comment]
Oh wow ... That was the desired behaviour. I'd update this PR ASAP. Thanks for the catch @andela-cdaniel

**Review by ChigboIO — COMMENTED** [review]
This is a good feature.
I don't know if it's the limitation of the library we're using, but the ``` for code block does not format and colour it very well.

What do we think of the preview background? I will suggest using something like the background of the trending questions side bar.

I will also suggest removing the **Preview** and putting the **(Learn markdown)** outside the preview box.

Nice work man :+1:

**Comment by andela-iamadi** [comment]
@andela-echigbo Modified accordingly... thanks for the details .. check out the edits 🙏

File: app/js/components/questions/new/Index.react.js [hunk 1]
```diff
@@ -49,10 +49,13 @@ class NewQuestion extends React.Component {
 
    render() {
      return (
-       <NewQuestionPage
-         onUpdateQuestionState={this.onUpdateQuestionState}
-         onUpdateTags={this.onUpdateTags}
-         onSubmitClick={this.onCreateQuestion} />
+       <div>
```
Comment (abiodun0) [inline app/js/components/questions/new/Index.react.js]: ❓ Why do we need to put this in an extra div?

File: app/js/components/questions/new/NewQuestionPage.react.js [hunk 2]
```diff
@@ -60,8 +62,16 @@ const NewQuestionPage = ({
...
-                           content=""
+                         <textarea
+                           content={question.content}
+                           data-id="content"
+                           onChange={event => {
+                             onUpdateQuestionState('content',
+                             event.target.value);
+                           }} />
+
+                         {/*<TinyMCE
```
Comment (abiodun0) [inline app/js/components/questions/new/NewQuestionPage.react.js]: Let's endeavour to remove commented out code. We can always go back to the git history to retrieve them if ew need to

**Review by abiodun0 — APPROVED** [review]
Excellent work. 
For future imporovement
We should try and have variables that holds magic literals like 0.7871492em and 1em in the sass files
and something like $medium-height: 200px;
The margins and paddings px too should have their own variables so there would be uniformity when changing a particular variable on the app

**Comment by andela-iamadi** [comment]
Gracias @abiodun0 Would bear that in mind 👍

**Comment by andela-iamadi** [comment]
markdown demo

**Comment by andela-oakinniranye** [comment]
👍

**Review by andela-oakinniranye — APPROVED** [review]

**Review by ChigboIO — APPROVED** [review]
Looking good.
Nice work @andela-iamadi 
👏 👏

**Comment by andela-iamadi** [comment]
Thanks man @andela-echigbo You've got a great eye for design 👍 💯

**Pull request merged**